api/base/views.py

```diff
@@ -485,7 +489,10 @@ def get_queryset(self):
         auth = get_user_auth(self.request)
         node_pks = node.node_relations.filter(is_node_link=False).select_related('child')\
             .values_list('child__pk', flat=True)
-        return self.get_queryset_from_request().filter(pk__in=node_pks).can_view(auth.user, auth.private_link).order_by('-modified')
+        order = '-modified'
+        if self.request.query_params.get('_order', None) == 'component_order':
```

Comment:
I think this is making it a little more difficult than it has to be. Ideally instead of `/?_order=component_order` you should use DRF existing `sort` param and `/?sort=_order` . I would look and `OSFOrderingFilter` and potentially add a `_order` field to the serializer.